app/models/input.rb

```diff
@@ -20,21 +20,21 @@
 class Input < Board
 
   def get_methods
-    { run: "default method" }
+    { run: "activate" }
+  end
+
+  def broadcast
```

Comment:
pai pt ca nu voiam ca metoda de broadcast sa trimita date pe canal inapoi la input ca nu vad rostul. Adica aia din board e tehnic mai mult pt Output type of devices.